### PR TITLE
fix(bundler): fix an issue with requiredExternals

### PR DESCRIPTION
**Pull request by JamesSingleton**
There was an issue where if a library like iguazu was listed ahead of a library
like iguazu-rpc, iguazu was getting bundled instead of iguazu-rpc.

### CODE REVIEW

File: packages/one-app-bundler/__tests__/utils/extendWebpackConfig.spec.js [hunk 1]
```diff
@@ -112,12 +112,13 @@ describe('extendWebpackConfig', () => {
     getConfigOptions.mockReturnValueOnce({ requiredExternals: ['ajv', 'lodash'] });
     const result = extendWebpackConfig(originalWebpackConfig);
     const { rules } = result.module;
+    console.log('Rules: ', rules);
```
Comment (Francois-Esquire) [inline packages/one-app-bundler/__tests__/utils/extendWebpackConfig.spec.js:115]:
```suggestion
```

File: packages/one-app-bundler/utils/extendWebpackConfig.js [hunk 2]
```diff
@@ -118,6 +118,8 @@ function extendWebpackConfig(webpackConfig, bundleTarget) {
     });
   }
 
+  // console.log(JSON.stringify(merge(webpackConfig, customWebpackConfig)));
```
Comment (Francois-Esquire) [inline packages/one-app-bundler/utils/extendWebpackConfig.js:121]:
```suggestion
```

Comment (JAdshead) [inline packages/one-app-bundler/utils/extendWebpackConfig.js:121]: remove comment

**Comment by JamesSingleton** [comment]
> Thanks for fixing this. wish we had a test to catch this type of obscure issues in the future, but well... webpack config is a maze

And we use the bundlers package.json to test it.

**Pull request merged**